test/unit/lib/classes/plugin-manager.test.js

```diff
@@ -75,14 +74,14 @@ class PromisePluginMock {
   }
 
   functions() {
-    return new BbPromise((resolve) => {
+    return new Promise((resolve) => {
```

Comment:
@sleepwithcoffee goal of this issue, was to get rid of also `new Promise` calls

Reply:
noted, I will work on it